pkgs/development/python-modules/h3/default.nix

```diff
@@ -1,35 +1,37 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , buildPythonPackage
-, cmake
 , fetchPypi
 , h3
-, python
+, scikit-build
+, cmake
+, numpy
+, cython
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "h3";
-  version = "3.7.0";
+  version = "3.7.1";
+  format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "cd27fc8ecd9183f93934079b7c986401f499030ff2e2171eace9de462fab561d";
+    sha256 = "16pa023rdrw5iylky0y1mycz8pkfdwchr531w7grpyn0n06ym5w0";
   };
 
-  patches = [
-    ./disable-custom-install.patch
-    ./hardcode-h3-path.patch
-  ];
```

Comment:
Does this mean the python package will no longer use the h3 package and instead build its own?

Reply:
Yes, I think that's correct. These patches definitely don't apply cleanly anymore, so I didn't bother.

Reply:
Unfortunately, I don't think that this is the correct path forward for this fix then. We should not be allowing the Python version to compile its own when we have a package for it. IMO it's best to fix these patches (feel free to use `substitute` instead of patches.)